test/autoinstall.js

```diff
@@ -0,0 +1,56 @@
+const assert = require('assert');
+const install = require('../src/utils/installPackage');
+const fs = require('fs');
+const rimraf = require('rimraf');
+const promisify = require('../src/utils/promisify');
+const primraf = promisify(rimraf);
+const ncp = promisify(require('ncp'));
+const inputDirPath = __dirname + '/input';
+
+describe('autoinstall', function() {
+  beforeEach(async function() {
+    // Setup (clear the input dir and move integration test in)
+    await primraf(inputDirPath, {});
+    await ncp(__dirname + '/integration/babel-default', inputDirPath);
+  });
+
+  it('should install lodash using npm and save dev dependency to package.json', async function() {
+    let pkgName = 'lodash';
+
+    await install({
+      dir: inputDirPath,
+      modules: [pkgName],
+      saveDev: true,
+      packageManager: 'npm'
+    });
+
+    let expectedModulePath = inputDirPath + '/node_modules/' + pkgName;
+    assert(fs.existsSync(expectedModulePath), 'lodash is in node_modules');
+
+    let pkg = fs.readFileSync(inputDirPath + '/package.json');
+    pkg = JSON.parse(pkg);
+    assert(pkg.devDependencies[pkgName], 'lodash is saved as a dev dep');
+  });
+
+  it('should install lodash using yarn and save dev dependency to package.json', async function() {
```

Comment:
Perhaps also add a test that tests the autoinstall from require.
For example just have a script like this that gets bundled and check if it's installed after test ran.
```
const _ = require('lodash');
// Do something with lodash to check later...
```

Reply:
👍

Reply:
This would be good to do in a followup PR. Maybe we can cover a few more of the different cases as well.